tools/scrape_supported_devices.py

```diff
@@ -247,7 +250,7 @@ def initargs():
   else:
     src = pathlib.Path(ARGS.directory) / "src"
   if not src.is_dir():
-    errorexit("Directory not valid: {}".format(str(src)))
+    errorexit(f"Directory not valid: {str(src)}")
```

Comment:
```suggestion
    errorexit(f"Directory not valid: {src!s}")
```
The `!s` is not required, but will ensure it's forced to a string (and not use the `__repr__`)

Reply:
Done